docs/en/setup/backend/backend-receivers.md

```diff
@@ -128,3 +129,20 @@ receiver-oc:
     gRPCHost: ${SW_OC_RECEIVER_GRPC_HOST:0.0.0.0}
     gRPCPort: ${SW_OC_RECEIVER_GRPC_PORT:55678}
 ```
+
+## Kafka receiver
+
+Kafka Receiver supports to receive `TraceSegment`, `JVMMetrics` and others of Agent reported as gRPC Receiver.
+Currently, it has to connect to the whole topic or single partition of topic when OAP is in cluster mode.
+And we have to configure **Zookeeper** as the OAP cluster management tool.
```

Comment:
I suggest not to use too many different terms when you're saying the same thing

```suggestion
And we have to configure **Zookeeper** as the OAP cluster coordinator.
```

Reply:
Why we need a Zookeeping to be OAP cluster management? Which parts of codes using the this? 
This fetcher is only taking responsibility for itself, the coordinator is for distributed aggregation. If the user wants Zookeeper for Kafka, and another coordinator, such as k8s, for OAP, I can't see the blocks. Is this?